gridwords.py

```diff
@@ -96,7 +97,7 @@ def setText(self, text):
     def setClueIndex(self, index):
         self.clue_index.set(str(index))
 
-    def onClick(self):
+    def onClick(self, args=None):
```

Comment:
Since the function bound to `Cell.clue_label` expects an extra argument (which we happen to be ignoring) I had to declare a default `args` value in the `Cell.onClick()` method.